src/biokbase/narrative/tests/test_narrative_logger.py

```diff
@@ -97,7 +94,7 @@ def assert_log_msg(self, msg, event, narrative, version):
         data = json.loads(msg)
         self.assertEqual(data["type"], "narrative")
         self.assertEqual(data["user"], "anonymous")
-        self.assertEqual(data["env"], "dev")
+        self.assertEqual(data["env"], "ci")
```

Comment:
the docker container defaults to CI, so when running the backend tests within the docker container, it makes sense to change these tests to expect CI.

Reply:
Nice